p/themes/base-theme/frss.css

```diff
@@ -115,6 +115,10 @@ h3 {
 	display: none;
 }
 
+.no-desktop {
```

Comment:
Is this a phrasing that's already used elsewhere? If this is brand new something like d-l-none/d-large-none or d-wide-none would be more neutral. But looking at the diff I'd call this d-none/display-none or something along those lines and the other d-mobile/small/whatever-unset.

Reply:
`.no-mobile` is already used. That is why I introduced `.no-desktop`

Reply:
While I don't necessarily like that one either then, it does work better since in practical terms it doesn't implicitly seem to suggest that one might be trying to exclude tablets.

In short, I want it to be as clear as possible while reading without having to look up whether it actually just means large/wide or if it actually means desktop. That's not always possible, but in this case it seems fairly easily attainable using something like the `.d-none` combined with `.d-sm-unset` pair I suggested above.

@Alkarex what do you think?

Reply:
Right now we have just 2 screen sizes in the CSS: Screens smaller than `840px` (`@media (max-width: 840px)`) are the "mobile" screens.
`.no-mobile` is used to hide elements, that should not be shown on small screens.
The code has already an implicite `.no-desktop`, but it has not that CSS class.

Reply:
Right, but it's a neutral large/wide, not "desktop". If you write "desktop" it makes you think of things like `@media (hover: hover)` vs `@media (hover: none)`.

Reply:
Like I said, something involving for example `.d-none`, `.d-sm-unset` and/or `.d-lg-none`.

Reply:
what does `d` and `sm` and `lg` mean?

Reply:
In the context of utility classes d is a common abbreviation for display.

So something like includes for example `.display-none` and `display-small-unset`.

Reply:
it would be a complete new coding style for the CSS in FreshRSS

Reply:
It's not quite clear to me what you're referring to. My very argument is that something along the lines of `small` and `large` constitutes neutral names mostly free from the potentially misleading assumptions that come along with `mobile` and particularly the proposed `desktop`. So yes, it'd be new to the 4+2 lines of code where `.no-mobile` is currently used.

To perhaps illustrate the problem better, consider calling it `.no-tv` instead.

Reply:
I changed it to `.only-mobile`

Reply:
Well… okay, lol.